README.md

```diff
@@ -15,6 +15,7 @@ It follows the Client credentials grant flow from the OAuth2.0 Framework [see he
 - AWS SAM
 - node v.20
 - npm
+- nvm
```

Comment:
Nit: Technically not a dependency unless we're planning on managing multiple node versions in our scripts. It would be the developers responsibility to ensure they meet the `node v.20` [sic] requirement 

(Also other version managers are [available](https://github.com/Schniz/fnm))

Reply:
Agree with this - it is just for developer convenience if we use nvm (but good thought on even thinking about documentation).

What we **do** need is an engines property in our package.json to enforce the use of a certain node version (both locally and in the pipelines)

Reply:
Will remove this from the readme. @sandy-may re your point about the engines, did you see my notes on that? I think if we added the engines property there would have to be an instruction to devs to update their own config to force the correct node version

Reply:
Shouldn't we always be forcing devs to user the correct node version?